virttest/env_process.py

```diff
@@ -765,6 +766,19 @@ def preprocess(test, params, env):
     logging.debug("KVM userspace version: %s" % kvm_userspace_version)
     test.write_test_keyval({"kvm_userspace_version": kvm_userspace_version})
 
+    # Checking required qemu, if not satisfied, cancel test
```

Comment:
it looks duplicate with above changes,  please try to remove duplicate.

Reply:
Many thanks for the review.
There is a plan to re-design the check procedures in `preprocess` and `postprocess`, this is sort like a temporary implemantation  to check the version of qemu and kernel.
There will be a new patch to handle this.